readme.md

```diff
@@ -89,6 +89,34 @@ Default: [User's downloads directory](http://electron.atom.io/docs/api/app/#appg
 
 Directory to save the file in.
 
+#### errorTitle
+
+Type: `string`<br>
+Default: `Download Error`
+
+The title of the error dialog. Can be customised for localization.
+
+#### errorMessage
+
+Type: `string`<br>
+Default: `The download of FILENAME was interrupted`
+
+The message of the error dialog. `FILENAME` is replaced with the name of the actual file. Can be customised for localization.
+
+#### onProgress
+
+Type: `function`<br>
+Default: `undefined`
+
+An optional callback to receive a ratio between `0` and `1` with the state of the current download.
+
+#### onComplete
```

Comment:
I don't really see the point of this one. You can just use the `.then()` on the promise to achieve the same.

Reply:
@sindresorhus Could `onComplete` be reconsidered? I use one `electronDl()` call for all windows and I need this option. I could reconfigure how downloads work to do downloads manually but that seems a bit heavy handed.

I would be happy to PR this.